### PR TITLE
Update nextClosestTime.py

### PR DESCRIPTION
**Pull request by kyuhankimsmc**
Added a missing import statement

### CODE REVIEW

**Review by justins2014 — COMMENTED** [review]
It is good to shorten the module name in the line 8.

**Comment by kyuhankimsmc** [comment]
It looks ok for me

File: nextClosestTime.py [hunk 1]
```diff
@@ -1,3 +1,5 @@
+import itertools
```
Comment (justins2014) [inline nextClosestTime.py]:
```suggestion
import itertools -> from itertools import product
```

**Review by justins2014 — CHANGES_REQUESTED** [review]

File: nextClosestTime.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import itertools
+import itertools -> from itertools import product
```
Comment (justins2014) [inline nextClosestTime.py]: import itertools

**Review by justins2014 — CHANGES_REQUESTED** [review]

**Review by justins2014 — APPROVED** [review]

**Pull request merged**